assets/package.json

```diff
@@ -8,6 +8,9 @@
     "npm": "^9.5.1"
   },
   "dependencies": {
+    "phoenix": "^1.7.7",
+    "phoenix_html": "^3.3.1",
+    "phoenix_live_view": "^0.19.5",
```

Comment:
If I remember right, with `esbuild`, we do not have to explicitly maintain JS dependencies.

Reply:
We could use the Javascript deps inside the `deps` folder built-in with the elixir packages but this means we have to change how we build our Docker application. This could be done in another pull request!